internal/xds/translator/cluster.go

```diff
@@ -106,6 +107,13 @@ func buildXdsCluster(args *xdsClusterArgs) *clusterv3.Cluster {
 		PerConnectionBufferLimitBytes: buildBackandConnectionBufferLimitBytes(args.backendConnection),
 	}
 
+	// 50% is the Envoy default value for panic threshold. No need to explicitly set it in this case.
```

Comment:
why is this set twice - here and below ?

Reply:
Good question, error while refactoring